src/validation.ts

```diff
@@ -669,6 +673,40 @@ export class OpenAPIValidator<D extends Document = Document> {
     return validators;
   }
 
+  /**
+   * Removes binary properties from the required array in JSON schema, since they cannot be validated.
+   *
+   * @param {OpenAPIV3_1.SchemaObject} schema
+   * @memberof OpenAPIValidator
+   */
+  private removeBinaryPropertiesFromRequired(schema: OpenAPIV3_1.SchemaObject): void {
+    if (typeof schema !== 'object' || !schema?.required) {
+      return;
+    }
+
+    if (schema.properties && schema.required && Array.isArray(schema.required)) {
+      const binaryProperties = Object.keys(schema.properties).filter((propName) => {
+        const prop: OpenAPIV3_1.SchemaObject = schema.properties[propName];
+        return prop && prop.type === 'string' && prop.format === 'binary';
+      });
+
+      if (binaryProperties.length > 0) {
+        schema.required = schema.required.filter((prop: string) => !binaryProperties.includes(prop));
+      }
+    }
+
+    // Recursively process nested objects and arrays
+    if (schema.properties) {
+      Object.values(schema.properties).forEach((prop) => this.removeBinaryPropertiesFromRequired(prop));
+    }
+
+    ['allOf', 'anyOf', 'oneOf'].forEach((key) => {
+      if (Array.isArray(schema[key])) {
+        schema[key].forEach((subSchema: any) => this.removeBinaryPropertiesFromRequired(subSchema));
+      }
+    });
+  }
```

Comment:
I don't follow what this has to do with supporting `multipart/form-data` or `application/x-www-form-urlencoded`. Is this intended to be part of this change?